src/main/java/uk/gov/hmcts/reform/professionalapi/controller/LegacyPbaController.java

```diff
@@ -63,19 +78,19 @@
     )
     @ResponseBody
     public ResponseEntity<LegacyPbaResponse> retrievePbaAccountsByEmail(@PathVariable("email") @NotBlank String email) {
-
         List<String> pbaNumbers;
-        ProfessionalUser professionalUser =  professionalUserService.findProfessionalUserByEmailAddress(RefDataUtil.removeEmptySpaces(email));
-        if (professionalUser == null) {
+        ProfessionalUser professionalUser = professionalUserService.findProfessionalUserByEmailAddress(RefDataUtil.removeEmptySpaces(email));

```

Comment:
there should be no functional change as this JIRA is for only Swagger related changes

Reply:
There have been no functional changes, these are formatting changes

Reply:
Ideally you should not change formatting for which you are not changing any code. Next time you can take care of this.

Reply:
Okay, noted!